sdcm/utils/aws_region.py

```diff
@@ -172,41 +221,67 @@ def create_sct_internet_gateway(self):
-            f"in {self.region_name}: {existing_rts}!"
+            f"More than 1 Route Table with {sct_route_table_name} found in {self.region_name}: {existing_rts}!"
         return self.resource.RouteTable(existing_rts[0]["RouteTableId"])  # pylint: disable=no-member
 
-    def configure_sct_route_table(self):
+    def create_sct_subnet_route_table(self, subnet_name: str, index: int = None):
+        subnet_route_table = self.sct_route_table_name(index=index)
+        LOGGER.info("Configuring SCT subnet Route Table %s for subnet %s...", subnet_route_table, subnet_name)
+        if sct_route_table := self.sct_route_table(index=index):
+            LOGGER.warning("Route Table '%s' already exists! Id: '%s'.",
+                           subnet_route_table, sct_route_table.route_table_id)
+            return
+
+        self.client.create_route_table(DryRun=False,
+                                       VpcId=self.sct_vpc.vpc_id,
+                                       TagSpecifications=[
+                                           {'ResourceType': "route-table",
+                                            'Tags': [{'Key': 'Name',
+                                                      'Value': subnet_route_table
+                                                      }]
+                                            }],
+                                       )
+        LOGGER.info("Setting routing of all outbound traffic via Internet Gateway...")
+        route_table = self.sct_route_table(index=index)
+        route_table.create_route(DestinationCidrBlock="0.0.0.0/0",
+                                 GatewayId=self.sct_internet_gateway.internet_gateway_id)
+        route_table.create_route(DestinationIpv6CidrBlock="::/0",
+                                 GatewayId=self.sct_internet_gateway.internet_gateway_id)
+
+    def configure_sct_route_table(self, index: str = ""):
         # add route to Internet: 0.0.0.0/0 -> igw
         LOGGER.info("Configuring main Route Table...")
-        if self.sct_route_table:
+        if sct_route_table := self.sct_route_table(index=index):
             LOGGER.warning("Route Table '%s' already exists! Id: '%s'.",
-                           self.SCT_ROUTE_TABLE_NAME, self.sct_route_table.route_table_id)
+                           self.sct_route_table_name(index=""), sct_route_table.route_table_id)
         else:
             route_tables = list(self.sct_vpc.route_tables.all())
             assert len(route_tables) == 1, f"Only one main route table should exist for {self.SCT_VPC_NAME}. " \
                                            f"Found {len(route_tables)}!"
             route_table: EC2ServiceResource.RouteTable = route_tables[0]
-            route_table.create_tags(Tags=[{"Key": "Name", "Value": self.SCT_ROUTE_TABLE_NAME}])
+
+            route_table.create_tags(Tags=[{"Key": "Name", "Value": self.sct_route_table_name(index="")}])
```

Comment:
`index` is not passed to method, hardcoded value is

Reply:
fixed